capa/features/extractors/ghidra/global_.py

```diff
@@ -10,7 +10,7 @@
 logger = logging.getLogger(__name__)
 
 def extract_os() -> Iterator[Tuple[Feature, Address]]:
-    format_name: str = currentProgram.getExecutableFormat()
+    format_name: str = currentProgram.getExecutableFormat() # currentProgram: static Ghidra variable
```

Comment:
Does this pass black formatting?

Reply:
I've gone ahead and linted the entire repo to the capa config's specifications. Please see 536677a505a6697ba3ca01998fe39cc78fe3104b and additional commits.